programs/server/uuid

```diff
@@ -0,0 +1 @@
+2e83a9e9-1c5d-4ae0-84d8-cbebb06b63f5
```

Comment:
Wrong files committed.

Reply:
I have no idea, when and why this string was inserted in programs/server/uuid.  Should I delete this file?

Reply:
Yes, It's created by clickhouse server when you start it. You can delete these files from the PR `programs/server/coordination/logs/changelog_1_100000.bin.zstd`, `programs/server/coordination/state`, `programs/server/uuid`